alert-policies/apache-flink/TaskManagerDirectMemoryUsed.yml

```diff
@@ -0,0 +1,26 @@
+name: Task Manager Direct Memory Used
+
+description: |+
+  This alert is triggered when the task manager direct - memory used is higher than 100mb during 5 minutes.
```

Comment:
I feel there is no need of using '-' in the text. Request you to kindly remove it.

Reply:
Required changes are done.